cli/status.go

```diff
@@ -90,7 +90,7 @@ func newAPIReachabilityStatus() apiReachabilityStatus {
 		Services: []*apiPing{
 			{Service: "GitHub", URL: "https://api.github.com"},
 			{Service: "Exercism", URL: "http://exercism.io/api/v1"},
-			{Service: "X-API", URL: "http://x.exercism.io"},
+			{Service: "X-API", URL: "http://x.exercism.io/ping"},
```

Comment:
I just realized—we're going to be killing off the x-api (yay).

So we will only have one exercism API to test against. The URL is `https://api.exercism.io/v1/ping` right now. I don't know how much of the `config.APIConfig` is present in the codebase at the moment, but we can rework this to use load up that config, which has the `BaseURL` (which I think is https://api.exercism.io/v1 right now), and we can add a `ping` endpoint to the default endpoints.